src/main/java/org/apache/hadoop/hdfs/server/namenode/NNLoader.java

```diff
@@ -2142,7 +2142,7 @@ public void initReloadThreads(ExecutorService internalService) {
               while (true) {
                 // Reload Keytab every 10 minutes.
                 try {
-                  Thread.sleep(10 * 60 * 1000L);
+                  Thread.sleep(conf.getSuggestionsReloadSleepTime());
```

Comment:
Let's not change the keytab reload time too. We can leave this hard-coded for now.